src/helm/benchmark/run_specs/melt_run_specs.py

```diff
@@ -291,3 +293,117 @@ def get_math_spec(
         + get_generative_harms_metric_specs(),
         groups=["melt"] + groups,
     )
+
+
+@run_spec_function("melt_text_classification_vsmec")
+def get_melt_text_classification_vsmec_spec(fewshot: bool = False) -> RunSpec:
```

Comment:
In general, you can already use the `max_train_instances=0` run expander, so you don't need a `fewshot` parameter.

E.g. `melt_text_classification_vsmec:model=text,max_train_instances=0`